src/components/MenuBox.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+	<div class="container" >
+		<img :src="require(`@/assets/${imgPath}`)">	
```

Comment:
is `require` works here?
what about remoe it and just using image path like this code?
https://github.com/trade-dog/web_front/blob/5c02624fa699a9a4cefed686d9a4072cfcfc67b2/src/components/TrendCard.vue#L28

Reply:
Thank you for suggestion. 👍 
`require` works here.
I tested `<MenuBox imgPath="gubean.jpg"...></MenuBox>`
with this code `<img :src="'@/assets/' + imgPath">` in views.  
Unfortunately, this code did not work.

Reply:
Okay. I'll check about about it after merged into `Menu` branch at this repo.
This issue will be treated another PR before merged into master branch.